react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -91,6 +101,24 @@ class OverflowMenu extends Component<Props> {
     }
 }
 
-OverflowMenu_ = connect()(OverflowMenu);
+/**
+ * Maps (parts of) the redux state to the associated props for the
+ * {@code OverflowMenu} component.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {{
+ *     _ccButtonVisible: boolean
+ * }}
+ * @private
+ */
+export function _mapStateToProps(state: Object): Object {
```

Comment:
The existing buttons own their state, they don't rely on their parent telling them their state. The buttons were also designed to cooperate in the display that their parent specifies. So OverflowMenu until this PR was just a container of buttons. If OverflowMenu had chosen to do like Web and have the parent take care of the state, then it would have already exploded into a very big mapStateToProps like Web.

Reply:
I'll change it